src/libtorch.cc

```diff
@@ -1213,11 +1305,31 @@ ModelInstanceState::ProcessRequests(
   }
 
   uint64_t compute_end_ns = 0;
-  RESPOND_ALL_AND_SET_TRUE_IF_ERROR(
-      responses, request_count, all_response_failed,
-      RecordBackendTimestamp(
-          &compute_end_ns,
-          reinterpret_cast<void*>(&compute_output_start_event_)));
+  std::atomic<uint64_t> compute_output_start = 0;
+
+  if ((Kind() == TRITONSERVER_INSTANCEGROUPKIND_MODEL) && (device_cnt_ > 0)) {
```

Comment:
I think for KIND_MODEL and device_cnt == 0 the stats are not handled properly.

Reply:
I think for KIND_MODEL and device_cnt == 0, it will just go to the else case and capture the `compute_end_ns` in `RecordBackendTimestamp`.

Reply:
sorry, I think I now understand how it is handled. Resolving.